src/sysusers/sysusers.c

```diff
@@ -1720,15 +1727,14 @@ static int parse_line(const char *fname, unsigned line, const char *buffer) {
                 if (r < 0)
                         return log_oom();
 
-                i = new0(Item, 1);
+                i = item_new(action[0], resolved_name);
```

Comment:
i'd just pass ADD_USER as first arg. passing the variable here is just needlessly indirect.